src/Configuration/Utils/FromJsonWithDef.hs

```diff
@@ -209,21 +208,3 @@ instance (FromJSON a, FromJsonWithDef a) ⇒ FromJsonWithDef [a] where
     Just v → parseJsonWithDef d v
   where
     err = fail $ "missing property " ⊕ T.unpack js
-
--- | This operator is an alternative for '($)' with a higher precedence which
--- makes it suitable for usage within Applicative style funtors without the need
--- to add parenthesis.
---
--- The hex value of the UTF-8 char × is 0x00d7
---
--- In vim type: @Ctrl-V u 00d7@
---
--- You may also define a key binding by adding something like the following line
--- to your vim configuration file:
---
--- > iabbrev <buffer> >< ×
---
-(×) ∷ (a → b) → a → b
-(×) = ($)
-infixr 5 ×
-{-# INLINE (×) #-}
```

Comment:
Let's not remove, but just deprecate this for now. Removing would break downstream code.